### PR TITLE
Added rubber log variants to #minecraft:logs_that_burn

### PR DESCRIPTION
**Pull request by a-a-GiTHuB-a-a**
## What
This simply adds the rubber log variants to the Minecraft tag `#minecraft:logs_that_burn`.

## Implementation Details
This adds three lines to one JSON file. Not much to discuss here.

## Outcome
This will make the rubber log's smelting properties more in-line with the other Minecraft logs.

### CODE REVIEW

**Review by screret — CHANGES_REQUESTED** [review]
Do it with the block definition in GTBlocks. This way the change will be overwritten.
then run the runData task in gradle.

**Comment by a-a-GiTHuB-a-a** [comment]
I added the tags in the registrate.

File: src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ public static ItemColor leavesItemColor() {
             .initialProperties(() -> Blocks.STRIPPED_SPRUCE_LOG)
             .lang("Stripped Rubber Log")
             .blockstate((ctx, provider) -> provider.logBlock(ctx.get()))
-            .tag(BlockTags.MINEABLE_WITH_AXE)
+            .tag(ItemTags.LOGS_THAT_BURN, BlockTags.MINEABLE_WITH_AXE)
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java]: that's an item tag so put it on the item, not the block

File: src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java [hunk 2]
```diff
@@ -1082,7 +1082,7 @@ public static ItemColor leavesItemColor() {
             .lang("Rubber Wood")
             .blockstate((ctx, provider) -> provider.axisBlock(ctx.get(),
                     provider.blockTexture(GTBlocks.RUBBER_LOG.get()), provider.blockTexture(GTBlocks.RUBBER_LOG.get())))
-            .tag(BlockTags.MINEABLE_WITH_AXE)
+            .tag(ItemTags.LOGS_THAT_BURN, BlockTags.MINEABLE_WITH_AXE)
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java]: ditto

File: src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java [hunk 3]
```diff
@@ -1091,7 +1091,7 @@ public static ItemColor leavesItemColor() {
             .lang("Stripped Rubber Wood")
             .blockstate((ctx, provider) -> provider.axisBlock(ctx.get(), provider.blockTexture(ctx.get()),
                     provider.blockTexture(ctx.get())))
-            .tag(BlockTags.MINEABLE_WITH_AXE)
+            .tag(ItemTags.LOGS_THAT_BURN, BlockTags.MINEABLE_WITH_AXE)
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java]: ditto

**Review by screret — CHANGES_REQUESTED** [review]

File: src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java (hunk 1 above)
Comment (a-a-GiTHuB-a-a) [reply src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java]: should be fixed

File: src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java [hunk 4]
```diff
@@ -1073,26 +1073,29 @@ public static ItemColor leavesItemColor() {
             .initialProperties(() -> Blocks.STRIPPED_SPRUCE_LOG)
             .lang("Stripped Rubber Log")
             .blockstate((ctx, provider) -> provider.logBlock(ctx.get()))
-            .tag(ItemTags.LOGS_THAT_BURN, BlockTags.MINEABLE_WITH_AXE)
+            .tag(BlockTags.MINEABLE_WITH_AXE)
             .simpleItem()
+            .tag(ItemTags.LOGS_THAT_BURN)
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java]: also nope. could you please, just please, actually pull the code and look at what's wrong?

**Review by screret — CHANGES_REQUESTED** [review]

File: src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java (hunk 4 above)
Comment (a-a-GiTHuB-a-a) [reply src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java]: should work now. hopefully. crossing all digits I can.

**Review by YoungOnionMC — APPROVED** [review]

**Review by screret — APPROVED** [review]

**Pull request merged**